src/types/sized_string.c

```diff
@@ -21,10 +21,10 @@ void free_sized_string(sized_string_t * string) {
 }
 
 sized_string_t copy_str_to_sized_string(char * str, size_t len) {
```

Comment:
could be simpler ? by reusing new_sized_string function and strncpy after

Reply:
Yeah you're right. I will change that

Reply:
I changed it, can you check ?